cmd/bnsd/x/nft/blockchain/handler_test.go

```diff
@@ -188,3 +193,50 @@ func TestQueryTokenByName(t *testing.T) {
 	require.NoError(t, err)
 	_ = x // todo verify stored details
 }
+
+func TestDeterministicStorage(t *testing.T) {
+	var helpers x.TestHelpers
+	for i := 0; i < 10000; i++ {
+		t.Run(fmt.Sprint(i), func(t *testing.T) {
+			db := iavl.MockCommitStore()
+			bucket := blockchain.NewBucket()
+			tickerBucket := ticker.NewBucket()
+			var owner weave.Address
+			owner.UnmarshalJSON([]byte("4orppuuU/Ii3PrfL1rh7+T65vvA="))
+			handler := blockchain.NewIssueHandler(helpers.AlwaysTrueAuthenticator(), nil, bucket, tickerBucket.Bucket)
+
+			// when
+			rawJson := `
+ 	{
+    "owner": "4orppuuU/Ii3PrfL1rh7+T65vvA=",
```

Comment:
You could also just pull the raw signed transaction from the block and post that as well, right?

I know, quite opaque but closer to reality. 

Also, maybe it needs a series of transactions?